src/components/QuickLinksDialog/components/LinksList/index.js

```diff
@@ -13,6 +13,22 @@ function ListItemLink(props) {
   return <ListItem component="a" {...props} />;
 }
 
+// Currently all the URLs in Links tab are hardcoded in this file
+// @example
+// <List component="nav" subheader={<ListSubheader component="div">Subheader Title goes here</ListSubheader>}>
+// <ListItemLink target="_blank" rel="noopener" href="https://www.PutYourURL.com">
+//   <ListItemIcon>
+//     <img
+//       className="css, if we have one"
```

Comment:
This commented out code from the QuickLinks Dialog doesn't seem useful. Can we remove it?

Reply:
Since it is hardcoded data, I thought a template could be helpful. If it seems unnecessary, we can remove it.